contributed_definitions/nyaml/NXbeam.yaml

```diff
@@ -1,70 +1,84 @@
-doc: |
-  Properties of the neutron or X-ray beam at a given location. 
+category: base
+doc: |   
+  Properties of the neutron or X-ray beam at a given location.
         
-  It will be referenced by beamline component groups within the :ref:`NXinstrument` group or by the :ref:`NXsample` group. 
-  Note that variables such as the incident energy could be scalar values or arrays. 
-  This group is especially valuable in storing the results of instrument simulations in which it is useful to specify the beam profile, 
-  time distribution etc. at each beamline component. 
-  Otherwise, its most likely use is in the :ref:`NXsample` group in which it defines the results of the neutron
+  It will be referenced by beamline component groups within the ':'ref':'`NXinstrument` group or by the ':'ref':'`NXsample` group.
```

Comment:
Having quotes around `:` may be a problem as the rst will maybe not parsed correctly.